src/components/05_pages/Content/Content.js

```diff
@@ -155,17 +159,47 @@ class Content extends Component {
         <Table>
           <TableHead>
             <TableRow>
-              {['Title', 'Content Type', 'Status', 'Updated', 'Actions'].map(
-                item => <TableCell key={item}>{item}</TableCell>,
-              )}
+              {[
+                'Title',
+                'Content Type',
+                'Author',
+                'Status',
+                'Updated',
+                'Actions',
+              ].map(item => <TableCell key={item}>{item}</TableCell>)}
             </TableRow>
           </TableHead>
           <TableBody>
             {this.props.contentList.map(
-              ({ type, attributes: { changed, nid, status, title } }) => (
+              ({
+                type,
+                attributes: { changed, nid, status, title },
+                relationships,
+              }) => (
                 <TableRow key={nid}>
                   <TableCell>{title}</TableCell>
                   <TableCell>{this.props.contentTypes[type].name}</TableCell>
+                  <TableCell>
+                    {this.props.includes['user--user'][
+                      relationships.uid.data.id
```

Comment:
This is kind of hard to read when we split this on multiple lines. Is this how our coding standards suggest us to do this?  🤔

Reply:
I guess we could increase the maximum line length in prettier. For me though it is not unreadable, but maybe that's simply because I'm quite used to prettier output.

Reply:
I don't really have an issue with this. Prettier is sometimes a bit of pain, but at least it makes things consistent.